backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -200,9 +200,53 @@ public Person addPatient(
       Boolean residentCongregateSetting,
       Boolean employedInHealthcare,
       String preferredLanguage) {
+    return addPatient(
+        _os.getCurrentOrganization(),
+        facilityId,
+        lookupId,
+        firstName,
+        middleName,
+        lastName,
+        suffix,
+        birthDate,
+        address,
+        telephone,
+        role,
+        email,
+        race,
+        ethnicity,
+        tribalAffiliation,
+        gender,
+        residentCongregateSetting,
+        employedInHealthcare,
+        preferredLanguage);
+  }
+
+  // IMPLICIT AUTHORIZATION: this is used for self-registration
```

Comment:
i would make it clear from the method name that this method is only supposed to be used for self-registration

Reply:
My intent is to move all three of these implicit self-registration methods into an entirely separate service, but would prefer to do so in a follow-up PR instead if that's ok by you